Feature/video surveys

henryrossiter opened the pull request:
Changed photo surveys to use native camera and started to implement video surveys on app-side.

```video``` prop is boolean, whether video responses are expected. When false, users only upload photos. this should be passed as a field of data from the admin panel. In ```app/containers/activity/screen/index.js```, I've set video to false temporarily, so that camera will function normally until video surveys are ready to go.

Also removed a few unused dependencies from package.json and removed instabug from the project.

shnizzedy commented:
Ref #63, https://github.com/orgs/ChildMindInstitute/projects/9#card-18421244

stufreen commented on app/containers/activity/screen/canvas/CameraInput.js:
```diff
@@ -23,100 +20,123 @@ const styles=StyleSheet.create({
+    color: '#00a30a',
+    fontSize: 60,
+  }
 });
```
Ideally we should pull the colors from a central place to encourage standardization and reuse.

Simplest way to do this would be to add the colors to a theme file in `app/themes` and import from there.

In the future we will need to take it a step further and make the themes switchable. There are a few options for this but we should probably discuss that as a team.

stufreen commented on app/containers/activity/screen/canvas/CameraInput.js:
```diff
@@ -23,100 +20,123 @@ const styles=StyleSheet.create({
+    ImagePicker.launchCamera(options, (response) => {
       if (response.didCancel) {
-        console.log('User cancelled image picker');
+        console.log('User cancelled video picker');
```
Remove debug code

stufreen commented on app/containers/activity/screen/canvas/CameraInput.js:
```diff
@@ -23,100 +20,123 @@ const styles=StyleSheet.create({
+        console.log('User cancelled video picker');
       } else if (response.error) {
-        console.log('Image Picker error: ', response.error);
+        console.log('Video Picker error: ', response.error);
```
Remove debug code

stufreen commented on app/containers/activity/screen/canvas/CameraInput.js:
```diff
@@ -23,100 +20,123 @@ const styles=StyleSheet.create({
+        console.log('Video Picker error: ', response.error);
       } else {
         let picSource = {uri: (Platform.OS == 'ios') ? response.uri.replace('file://','') : response.uri, filename: response.uri.split("/").pop()};
+        console.log("uri --- " + response.uri);
```
Remove debug code

stufreen commented on app/containers/activity/screen/canvas/CameraInput.js:
```diff
@@ -23,100 +20,123 @@ const styles=StyleSheet.create({
+    })
   }
-
+*/
```
Remove commented out code

stufreen commented on app/containers/activity/screen/canvas/CameraInput.js:
```diff
@@ -23,100 +20,123 @@ const styles=StyleSheet.create({
+  //props: video- indicates whether to allow user to upload videos. when videos is false, only photos can be uploaded
+  //take a new picture/video with native camera
+  take = () => {
+    let options = { mediaType: ((this.props.video) ? 'video' : 'photo'),
```
Use `const` instead of `let` wherever possible

henryrossiter commented:
Okay, fixed according to comments. Also removed the instabug build dependencies from ```android/```.

If this is merged, Lets leave #63 open for now, I think there are a few more packages that should be removed before that can be closed.
